api/nyc.config.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+module.exports = {
+  cwd: 'api',
+  include: [
+    'src/**/*.js'
+  ],
+  exclude: [
+    'tests/mocha/**/*.js',
+    'shared-libs/**/*.*',
+  ],
```

Comment:
Maybe I am missing something, but I do not see how this `exclude` config is adding anything.  `tests/` and (the non-existent) `shared-libs/` should not be picked up anyway because of the `include`.

Reply:
Done. I fiddled a bunch around I got it to never include extra files. I think the exclude was needed when I didn't use `cwd`.